src/commands/sovereignCloud/SovereignCloudListStep.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IAzureQuickPickItem } from "@microsoft/vscode-azext-utils";
+import { ConfigureSovereignCloudContext } from "./ConfigureSovereignCloudContext";
+
+export class SovereignCloudListStep extends AzureWizardPromptStep<ConfigureSovereignCloudContext> {
+    public async prompt(context: ConfigureSovereignCloudContext): Promise<void> {
+        const picks: IAzureQuickPickItem<string>[] = [
+            { label: 'Azure (Default)', data: '' },
+            { label: 'Azure China', data: 'ChinaCloud' },
+            { label: 'Azure US Government', data: 'USGovernment' },
+            { label: 'A custom Microsoft Sovereign Cloud', data: 'custom' }
```

Comment:
If they pick custom, should we take them to the setting page so they can actually configure it? We can't make this change later though.

Reply:
Yah we should change the custom since it will cause an error at the moment